sdk/search/tests.yml

```diff
@@ -0,0 +1,15 @@
+trigger: none
+
+jobs:
+  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+    parameters:
+      ServiceDirectory: search
+      DisableAzureResourceCreation: true
```

Comment:
This should only be set if there is a problem. The script will detect the presence of a `test-resources.json` and decide whether to create or not create resources.

Reply:
Will remove this line.

Reply:
This needs to be done by proper design. 
Search is creating resource group for every test class using management library.
Ideally, we can try to create one resource group per test run. I am trying to see if it works for search.

Reply:
Looking at the design it will create a resource group once per test run, this should be updated in the future to use an ARM generate resource group instead. I'll file an issue for that to track it.

Reply:
The live test will failed at provision test resource step by having this.